### ISSUE TITLE
Dimension Jumps Macro not completing last step

### ISSUE THREAD

**Issue by anchovy369** [MrVauxs/pf2e-jb2a-macros#68]
I'm noticing that when casting spells that use the Dimension Jumps macro (like Abundant Step, for example), the macro does not reset the token opacity to 1.  So, I have to do it manually.

**Comment by MrVauxs** [comment]
Abundant Step doesn't even turn you invisible, and I cannot reproduce this with Dimension Door which does make your token invisible.

Check console for errors when using the spells and provide the versions of the modules you use (Sequencer, Automated Animations, PF2e Animations).

**Comment by anchovy369** [comment]
Sequencer is 2.41, AA is 4.0.22, PF2e Animations is 2.7.2, and JB2a (Patreon Complete Collection) is 0.5.2.

I have attached the console errors with Abundant Step.  It may be an issue with Token Attacker or Token Mold but I get the same errors when I disabled them.

<img width="538" alt="Screen Shot 2022-12-12 at 4 58 24 PM" src="https://user-images.githubusercontent.com/120410994/207164860-3e832c3c-302b-4c34-b0da-c04faa860dbd.png">
<img width="537" alt="Screen Shot 2022-12-12 at 4 57 46 PM" src="https://user-images.githubusercontent.com/120410994/207164869-a3082eda-b3ba-4155-8773-2921c28a74af.png">
<img width="531" alt="Screen Shot 2022-12-12 at 4 57 28 PM" src="https://user-images.githubusercontent.com/120410994/207164872-1cfacc80-bd4a-47da-b2fd-4e4d4213cb90.png">
